f5router/f5router.go

```diff
@@ -192,7 +193,7 @@ func (r *F5Router) createHTTPVirtuals() {
 			"",
 			r.c.BigIP.Partitions[0],
 			va,
-			"http",
+			"tcp",
```

Comment:
Will these virtuals still get the /Common/http profile attached to them? The bigipconfigdriver has changed enough I'm not sure.

Reply:
Look at Lines 176-178 above -- "/Common/http" is added as the default profile if no other profiles are specified. I assumed that's the intended behavior.

Reply:
+1